app/state_machines/workflows/predicates.rb

```diff
@@ -114,7 +114,19 @@ def overturned_editing_enabled_and_responder_in_assigned_team?
   end
 
   def has_pit_extension?
-    @kase.extended_for_pit?
+    @kase.has_pit_extension?
+  end
+
+  def has_sar_deadline_extension?
+    @kase.try(:deadline_extended?)
+  end
+
+  def deadline_does_not_exceed_max_deadline?
+    @kase.try(:deadline_extendable?)
```

Comment:
Not sure about this try? In what circumstances might the @kase variable be nil?

Reply:
Good question @starswan - use of `try` as a defensive solution as `deadline_extendable?` is only available for SAR cases. Note added as per discussion.